irispy/iris_tools.py

```diff
@@ -87,8 +97,9 @@ def get_iris_response(pre_launch=False, response_file=None, response_version=Non
     Returns
     -------
     iris_response: `dict`
-        Various parameters regarding IRIS response.  The following keys:
-        date_obs: `datetime.datetime`
+        Various parameters regarding IRIS response or effective area structure.
+        Includes the following keys:
+        date_obs: `time.time`
         lambda: `astropy.units.Quantity`
```

Comment:
I dont know how we feel about all the output keywords being capitalized ... something to talk about in the future after this PR